Runtimes/Core/cmake/modules/DefaultSettings.cmake

```diff
@@ -46,6 +46,8 @@ if(APPLE)
   set(SwiftCore_THREADING_PACKAGE_default "DARWIN")
   set(SwiftCore_ENABLE_PRESPECIALIZATION_default ON)
   set(SwiftCore_CONCURRENCY_GLOBAL_EXECUTOR_default "dispatch")
+  # TODO we are not ready to enable it here
+  set(SwiftCore_ENABLE_DIRECT_RETAIN_RELEASE_default OFF)
```

Comment:
Can you add this to the other branches as well? This is the default value, so we need the specification.